wonderful_bing/wonderful_bing.py

```diff
@@ -37,8 +39,11 @@
 class WonderfulBing(object):
     def __init__(self, arguments):
         # Get all the information we need from this url, see issue#7
+        self.country = arguments['--country']
+        if not self.country:
+            self.country = 'en-US'
```

Comment:
When we don't specify the `country`, it will raise the `keyerror`, right? Also note that it should be `arguments['country']`

how about this one:

    self.country = arguments.get('country') or 'en-US'